resume-buddy/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -29,36 +29,22 @@
 public class CommentServlet extends HttpServlet {
 
   private List<Comment> comments;
-  static final int DEFAULT_COMMENTS_NUMBER = 5;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userType = (LoginServlet.getUserType()).toString().toLowerCase();
```

Comment:
I think you need to find a way for the user type to be part of the request. Getting it from the `LoginServlet` will run into issues when multiple people are using the site at the same time. User A can log in as reviewer, then user B can log in as reviewee, then user A can try to get comments, but the user type is incorrect.

Reply:
I will add this as a TODO and revisit it in my next PR